service_test.go

```diff
@@ -9,372 +9,239 @@ import (
 	"testing"
 )
 
-func setupServiceTestCase(t *testing.T) (*Client, func(t *testing.T)) {
-	t.Log("setup Service test case")
-
-	const (
-		UserName     = "test@aiven.io"
-		UserPassword = "testabcd"
-		AccessToken  = "some-random-token"
-	)
+var servicesHandler = &ServicesHandler{
```

Comment:
I'm not sure if this is a good idea, with this approach it's possible to use `servicesHandler` in other tests as well, which might be unexpected, WDYT?